asf_online_data_exploration/config/data_collection_parameters.py

```diff
@@ -0,0 +1,193 @@
+"""
+Script with parameters for data collection.
+"""
+
+# -----TWITTER DATA COLLECTION-----
+
+heating_technologies_ruleset_twitter = [
+    {
+        "value": '"heat pump" OR "heat pumps" OR #heatpump OR #heatpumps',
+        "tag": "general_heat_pumps",
+    },
+    {
+        "value": '"air source hp" OR "air source hps" OR "air-source hp" OR "air-source hps" OR #ashp OR #airsourcehp OR #airsourcehps OR #airsourceheatpump OR #airsourceheatpumps',
+        "tag": "ashp",
+    },
+    {
+        "value": '"ground source hp" OR "ground source hps" OR "ground-source hp" OR "ground-source hps" OR #gshp OR #groundsourcehp OR #groundsourcehps OR #groundsourceheatpump OR #groundsourceheatpumps',
+        "tag": "gshp",
+    },
+    {
+        "value": '"water source hp" OR "water source hps" OR "water-source hp" OR "water-source hps" OR #wshp OR #watersourcehp OR #watersourcehps OR #watersourceheatpump OR #watersourceheatpumps',
+        "tag": "wshp",
+    },
+    {
+        "value": '"air to air hp" OR "air to air hps" OR "air-to-air hp" OR "air-to-air hps" OR "air2air hp" OR "air2air hps" OR "a2a hp" OR "a2a hps" OR #atahp OR #airtoairhp OR #airtoairhps OR #airtoairheatpump OR #airtoairheatpumps OR #a2ahp OR #a2ahps OR #air2airhp OR #air2airhps OR #air2airheatpump OR #air2airheatpumps',
+        "tag": "atahp",
+    },
+    {
+        "value": '"air to water hp" OR "air to water hps" OR "air-to-water hp" OR "air-to-water hps" OR "air2water hps" OR "air2water hp" OR "a2w hp" OR "a2w hps" OR #atwhp OR #airtowaterhp OR #airtowaterhps OR #airtowaterheatpump OR #airtowaterheatpumps OR #a2whp OR #a2whps OR #air2waterhp OR #air2waterhps OR #air2waterheatpump OR #air2waterheatpumps',
+        "tag": "atwhp",
+    },
+    {
+        "value": '"hybrid hp" OR "hybrid hps" OR "bivalent hp" OR "bivalent hps" OR "warm air hp" OR "warm air hps"',
+        "tag": "heat_pump_others",
+    },
+    {
+        "value": '"gas boiler" OR "gas boilers" OR #gasboiler OR #gasboilers',
```

Comment:
Hey @ch-williamson, thanks so much!
For the prototype I really wanted to avoid false positives as much as possible, that's why I didn't add boiler. But I will add it now. If it proves too erroneous, we can simply filter the data with this tag :) 

Add these two rules:
```
[{"value":'(boiler OR boilers OR #boiler OR #boilers) -("boiler suit" OR "boiler suits")', "tag":"general_boilers"}, 
{"value":'"combi boiler" OR "combi boilers" OR #combiboiler OR #combiboilers', "tag":"combi_boilers"}]
```

and for The Guardian:
```
[{"value":'(boiler OR boilers) -("boiler suit" OR "boiler suits")', "tag":"general_boilers"}, 
{"value":'"combi boiler" OR "combi boilers"', "tag":"combi_boilers"}]
```
 Could you just double check the logic for me please? :)

Reply:
As per comment below, will just add the first rule, as results from the second will be included in the first :)

Reply:
Update: ended up keeping all, as "boiler"/"boilers" yields too many results!